app/templates/application.hbs

```diff
@@ -1,13 +1,17 @@
-{{page-title "My"}}
+{{page-title "My | Real Dev Squad"}}
 
-<Navbar @firstName={{@model.firstName}} @profilePictureURL={{@model.profilePictureURL}} @signOutHandler={{this.signOutHandler}} />
+<Navbar
+  @firstName={{@model.firstName}}
+  @profilePictureURL={{@model.profilePictureURL}}
+  @signOutHandler={{this.signOutHandler}}
+/>
 
-<div class="main-container">
+<div class='main-container'>
   {{outlet}}
 </div>
-
-<footer class="footer">Contents of this website are deployed from this
-  <a href="https://github.com/Real-Dev-Squad/website-my" target="_blank" rel="noopener noreferrer">
+<Holder />
+<footer class='footer'>Contents of this website are deployed from this
+  <a href={{this.url}} target='_blank' rel='noopener noreferrer'>
```

Comment:
Can we please make all quotes double quotes?
I think linting is not configured. Vinayak has already raised a issue, soon it will be fixed https://github.com/Real-Dev-Squad/website-my/issues/351

Reply:
Got it. Have corrected it in the latest commit.